source/MaterialXCore/Util.h

```diff
@@ -69,6 +69,10 @@ MX_CORE_API string createNamePath(const StringVec& nameVec);
 /// Given a name path, return the parent name path
 MX_CORE_API string parentNamePath(const string& namePath);
 
+/// Normalize a string containing a numeric value. This can
```

Comment:
I went with `normalizeNumericString` vs `normalizeValueString` since this specifically can only handle numeric values in a string. Also the precision allows for the conversion  of numbers to double to not be unbounded.

Reply:
I think I might still vote to name the function `normalizeValueString` because we may find need in the future to normalize other types of values - perhaps removing leading/trailing white space from filenames? or other future types we don't yet have.  

Especially if we relocate the type filtering logic inside this function, I think the more generic name makes more sense, and is a little more forward looking from an API design perspective.